app/views/content/blog/choosing-the-right-teacher-training-course-provider.md

```diff
@@ -0,0 +1,43 @@
+
+Your drive to become a teacher needs to inform your choice of provider. If you have a particular interest, make sure that the school placements you will be offered for your teaching practice will allow you to develop it. For example, there are providers that work with fully comprehensive and inclusive schools, faith-based schools, special schools, grammar schools, schools in challenging social or economic areas, rural schools and so on.
+
+You can find the available postgraduate teacher training courses in your area via the [Department for Education’s Find service](https://www.gov.uk/find-postgraduate-teacher-training-courses). Consider selecting 4-5 that you feel might be suitable for you, and then make contact with them to check that they are. Aim to shortlist these down to 3. 
+
+Alongside online research, think about calling or emailing these providers, or try to arrange a visit. Your questions may include:
+
+* How do you arrange the academic work alongside the practical teaching practice?
+* Who will my mentor be, and how often will I be able to meet them?
+* What schools will I have access to for teaching practice? 
+* How many schools will I be doing my teaching practice in and how long will each placement last? 
+* Will I be awarded a PGCE/PGDE in addition to QTS (Qualified Teacher Status)?
+* What are the starting dates of the course and when does it finish?
+* How many places are on offer, and how many other trainee teachers will I be studying alongside?
+
+>When I was considering becoming a teacher, I specifically wanted to work with children identified with Special Educational Needs and Disabilities (SEND). Having previously worked in the mental health sector, I also had a deep interest in holistic teaching and inclusivity. When choosing my teacher training course, this was at the top of my wish list for a provider to offer. I was fortunate to find a teacher training course provider that was able to allow me to practice in a mainstream school that also hosted a hearing impairment unit, and another school that offered support for children with a variety of learning needs. They were great at teaching me and letting me practice my classroom management skills early on into the course. The academic and theoretical input that I could put into practice on my school placements set me up well for my teaching career to come. Further, the academic assignments that I was required to complete for the PGCE element of the course was well integrated into my collection of evidence for meeting the QTS standards. I was able to research and understand ways to differentiate resources for children who were on the Autistic Spectrum, and I was able to create and implement a scheme of work to reflect on and evaluate for later use once qualified.
```

Comment:
This is a big block of text on the page. Perhaps we could split it up into paragraphs? I'm not sure how/where so won't leave a suggestion 🙈

Reply:
Yes I see what you mean - assuming it will still be clear that it's still a quote? I think start the new one at 'They were great at teaching me...'?

Reply:
Brilliant, thank you!

Reply:
Is there anything else needed from me on this one?